README.md

```diff
@@ -24,7 +24,7 @@ Get up and running in a few minutes with our drop-in turn-by-turn navigation `Na
 
 ## Requirements
 
-The Mapbox Navigation SDK and Core Navigation are compatible with applications written in Swift 4.2 or Objective-C in Xcode 10.0. The Mapbox Navigation and Mapbox Core Navigation frameworks run on iOS 9.0 and above.
+The Mapbox Navigation SDK and Core Navigation are compatible with applications written in Swift 5.0 or Objective-C in Xcode 10.2.1. The Mapbox Navigation and Mapbox Core Navigation frameworks run on iOS 9.0 and above.
 
 The last release compatible with Swift 3.2 was v0.10.1.
 
```

Comment:
When installing the library using Carthage, does the developer need to ensure that MapboxAccounts.framework is in the Embedded Binaries list? If so, we should also mention this requirement as a backwards-incompatible change in the changelog.

https://github.com/mapbox/mapbox-navigation-ios/blob/0fcaf83b1e5b2dbfba739ba23986bbcecb111914/README.md#L57